src/transition.js

```diff
@@ -83,7 +85,17 @@ export default async (options = {}) => {
   promises = []
   Object.keys(resolvers).forEach(index => {
     const resolver = resolvers[index]
-    const response = Promise.resolve(resolver({ route, params, location }))
+    const response = Promise.resolve(
+      resolver({
+        route,
+        params,
+        location,
+        // This has to be done because we get getState on initial match
+        // And store on subsequent client transitions
+        // At some point we could switch it to use store fully
+        getState: getState || (store || { getState: () => ({}) }).getState,
```

Comment:
`(store || { getState: () => ({}) })` is this for tests?

Reply:
This is for adding support for getState to our resolve functions

Reply:
Transition, match, and initialize all are initialized slightly differently